modular_zubbers/code/modules/mob/dead/dead.dm

```diff
@@ -0,0 +1,40 @@
+/mob/dead/get_status_tab_items()
+	.=..()
+	if(CONFIG_GET(flag/show_job_estimation))
+		. += get_job_estimation()
+
+/mob/dead/proc/get_job_estimation()
+	var/list/player_ready_data = list()
+	player_ready_data.Cut()
+	var/list/players = list()
+
+	for(var/mob/dead/new_player/player as anything in GLOB.new_player_list)
+		if(player.ready == PLAYER_READY_TO_PLAY)
+			players[player.key] = player
+
+		sortTim(players, GLOBAL_PROC_REF(cmp_text_asc))
+
+	for(var/ckey in players)
+		var/mob/dead/new_player/player = players[ckey]
+		var/datum/preferences/prefs = player.client?.prefs
+		if(!prefs)
+			continue
+
+		if(!prefs.read_preference(/datum/preference/toggle/ready_job))
+			continue
```

Comment:
```suggestion
		if(!prefs && !prefs.read_preference(/datum/preference/toggle/ready_job)
			continue
```

Reply:
this is wrong.
`if(!prefs || !prefs.read_preference(/datum/preference/toggle/ready_job)`
or
`if(!prefs?.read_preference(/datum/preference/toggle/ready_job)`

Reply:
Woop woop. Mb